client/my-sites/importer/importer-header.jsx

```diff
@@ -56,10 +58,22 @@ class ImporterHeader extends React.PureComponent {
 
 		if ( includes( [ ...cancelStates, ...stopStates ], importerState ) ) {
 			cancelImport( siteId, importerId );
+
+			this.props.recordTracksEvent( 'calypso_importer_main_cancel_import', {
+				importer_id: type,
```

Comment:
The blog_id is not coming through on these events -- should we include it?

Reply:
+1, I think `blog_id` would greatly improve usability of those events